app/src/main/java/com/orange/ods/app/ui/components/progress/ProgressLinear.kt

```diff
@@ -107,18 +110,25 @@ fun ProgressLinear() {
                         determinateProgressValue = DeterminateProgressTargetValue
                     }
                 }
+
                 CodeImplementationColumn {
-                    CommonTechnicalTextColumn(
-                        componentName = OdsComponent.OdsLinearProgressIndicator.name
-                    ) {
-                        if (type.value == ProgressCustomizationState.Type.Determinate) TechnicalText(text = " progress = $determinateProgressValue")
-                        if (hasLabel) TechnicalText(text = " label = \"$text\"")
-                        if (hasCurrentValue && type.value == ProgressCustomizationState.Type.Determinate) TechnicalText(text = " currentValue = \"${(DeterminateProgressTargetValue * 100).toInt()}\"")
-                        if (hasIcon) TechnicalText(text = " icon = painterResource(id = R.drawable.ic_arrow_down)")
-                    }
+                    ComposableCode(
```

Comment:
Horizontal padding is missing:

![Screenshot_20230419_110202](https://user-images.githubusercontent.com/13621389/233025136-5d949972-9f27-42b9-9188-544b47000c9d.png)